### DISCUSSION TITLE
Standard compile options for e_sqlite3

### DISCUSSION THREAD

**Discussion by meiaestro** [ericsink/SQLitePCL.raw#586] [Q&A] [answered]
Hi,

for our own .Net project we need to change the YYSTACKDEPTH compile option and thus need to integrate our own sqlite build into Microsoft.Data.Sqlite. Therefor we were switching to SQLitePCLRaw-Provider "dynamic_cdecl". To make sure we have a mostly identical behaviour, we would appreciate to know the standard compile options for the e_sqlite3 build.

Can you please provide us with these options.

Thanks a lot.
Joerg

**Comment by ericsink** [comment, marked as answer]
The build system for e_sqlite3 is over in my `cb` repo:

https://github.com/ericsink/cb/blob/c990759703be78795e3687daccd957fcd490f9a6/bld/cb.cs#L1237-L1248

**Reply by meiaestro** [reply]
Thank you!
